projects/project01/README.md

```diff
@@ -49,42 +49,54 @@ Your team will need to present a 10-min demo of the solution + 10 min questions
 - Technical solution working - 50%
 
 
-## Kick off
-- Project introduction
-- Decide on teams
+## Kick off (today!)
+
+- Project introduction (instructors)
+- Decide on teams (check with instructors)
+- Create Github repo (Automated) - Something like `202X-xxx-projectX-groupX`
+
+Your tasks:
+- [Create initial project backlog](../../scripts/projects/README.md)
 - Create channel in Slack with name (project_1_name1_name2_name3)
```

Comment:
is this name correct? Also, do they have access to create channels in slack?

Reply:
Yeah, I don't think they have access. I added this as an instructor task.